components/RoundSummary/RoundSummary.tsx

```diff
@@ -95,6 +93,20 @@ export const RoundSummary = ({
     },
   ];
 
+  const submissionsDisplayHeaders: {
```

Comment:
This is a Typescript thing right? From my understanding this will be used as a template to organized the data retrieved from Supabase?